setup.py

```diff
@@ -83,26 +83,29 @@ def check_pseudo_package(path):
             raise Exception("Please make sure pseudo-package %s is populated." % path)
 
 
-if __name__=="__main__":
+def pypi_version(pypi=False):
+    if not pypi: return
+    # Make sure to create these directories and populate them before upload
```

Comment:
Right now there is a manual step. Populating the pseudo packages is also a manual step.

I'm all in favor of replacing the manual steps with something else... as long as it works!